chains/evm/contracts/ccvs/CommitRamp.sol

```diff
@@ -119,7 +153,7 @@ contract CommitOnRamp is Ownable2StepMsgSender, BaseOnRamp {
   // ================================================================
 
   function getFee(
-    uint64, // destChainSelector
+    address, // originalCaller
```

Comment:
We would always need the dest chain selector in getFee

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/8f1762ef84e1348c20ab970fbc61f97ba2327bf7